packages/core/src/components/DatePicker/DatePicker.styles.tsx

```diff
@@ -28,28 +28,16 @@ export const styles: Partial<
   dropdown: {
     display: "block",
   },
-  panel: {
-    border: theme.datePicker.panelBorder,
-    backgroundColor: theme.datePicker.panelBackgroundColor,
```

Comment:
These three props can be removed from the theme:
- panelBorder
- panelBackgroundColor
- dropdownHeaderOpenBorder